db/migrate/20170820041112_create_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviews < ActiveRecord::Migration[5.0]
+  def change
+    create_table :reviews do |t|
+      t.string :text
+      t.boolean :verified, default: false
```

Comment:
in future add `null: false` to the fields that should never be `nil`. For example this field should be `true` or `false` and `null: false` will ensure that